### DISCUSSION TITLE
Add Toasts?

### DISCUSSION THREAD

**Discussion by Phosphorus-M** [mantinedev/mantine#1159] [Feature requests]
It's just a suggestion, but it would be great if Mantine had a toast component or system like the notification.

And don't worry the framework it's awesome!!
Long life to Mantine!

**Comment by rtivital** [comment]
Toasts are just notifications with different styles, and we already have notifications system.

**Reply by atumas-bananamilk** [reply]
But the Notifications don't stack, and we'd need to build everything around stacking / adding / removing / updating toasts. Completely different. Please get on it.

**Reply by kfrp** [reply]
They do stack.

**Reply by atumas-bananamilk** [reply]
Ah yes, found it, apparently “Notification” and “Notification System” are somehow 2 different things on Mantine.
